### DISCUSSION TITLE
How can I use Hermite? Why doesn't it work in that way?

### DISCUSSION THREAD

**Discussion by Moyiqi** [idaholab/moose#24066] [Q&A General] [answered]
Hello everyone! May I ask a question?
How can I use Hermite? Why doesn't it work in that way?
```
[Mesh]
  type = GeneratedMesh
  dim = 3
  nx = 15
  ny = 15
  nz = 15
  xmax = 0.06
  ymax = 0.06
  zmax = 0.06 
  # parallel_type = distributed
  elem_type= HEX27
[]

[Functions]
  [source]
    type = ParsedFunction
    expression = 600
  [../]
  [./dts]
    type = PiecewiseLinear
    y = '0 0.05 1 1'
    x = '0 1 2 50'
  [../]
[]


[Variables]
  [./temperature]
    order = THIRD
    family = HERMITE
    initial_condition = 295.0 # K
  [../]
  [./disp_x]
    order = THIRD
    family = HERMITE
  [../]
  [./disp_y]
    order = THIRD
    family = HERMITE
  [../]
  [./disp_z]
    order = THIRD
    family = HERMITE
  [../]
[]
````
![63617ee7768b1a9e57854ca30b2a18b](https://user-images.githubusercontent.com/59696390/231976914-aa39b979-c263-4057-98a4-3c8cdf388d71.png)

Lily

**Comment by GiudGiud** [comment, marked as answer]
Hello

Displacements are node displacements. So only a variable that is entirely defined by its nodal values is currently supported.
Support for the hermite basis is likely limited to niche applications. I m not aware of anyone using them

Guillaume

**Reply by Moyiqi** [reply]
Thank you so much!
 Lily
